device-types/Cisco/C9407R.yaml

```diff
@@ -3,19 +3,55 @@ manufacturer: Cisco
 model: Catalyst 9407R
 slug: cisco-c9407r
 part_number: C9407R
+comments: '[Cisco Catalyst 9400 Series Chassis Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9400-series-switches/nb-06-cat9400-ser-data-sheet-cte-en.html)'
 u_height: 10
-is_full_depth: false
-console-ports:
-  - name: con 0
-    type: rj-45
-power-ports:
-  - name: PS-1
-    type: iec-60320-c14
-    maximum_draw: 3200
-  - name: PS-2
-    type: iec-60320-c14
-    maximum_draw: 3200
-interfaces:
-  - name: GigabitEthernet0/0
-    type: 1000base-t
-    mgmt_only: true
+weight: 28.60
+weight_unit: kg
+is_full_depth: true
+airflow: right-to-left
+module-bays:
```

Comment:
Does the device itself not have a management port on it? It appears that the old definiction has a single interface, GigabitEthernet0/0 which was management only. It also appears to have has a console port as well. Are these not in the chassis by default?

I am also asking @DanSheps for further clarification

Reply:
Talked with Dan, he said they are on the supervisor. You can ignore this comment.